django-stubs/db/models/query.pyi

```diff
@@ -169,6 +169,9 @@ class _ValuesQuerySet(Generic[_T, _Row], Collection[_Row], Reversible[_Row], Que
     def latest(self, *fields: Any, field_name: Optional[Any] = ...) -> _Row: ...  # type: ignore
     def first(self) -> Optional[_Row]: ...  # type: ignore
     def last(self) -> Optional[_Row]: ...  # type: ignore
+    def distinct(self) ->  _ValuesQuerySet[_T, _Row]: ... # type: ignore
+    def order_by(self, *field_names: Any) -> _ValuesQuerySet[_T, Row]: ... # type: ignore
```

Comment:
Should it be `_Row`?

Reply:
Yep nice catch :)

It seems like typos like this just cause the return type to transform to `Any` based on the tests I wrote? I wasnt aware of this